src/pages/test-report/components/vehicle-check/vehicle-check.ts

```diff
@@ -0,0 +1,52 @@
+import { Component, OnInit } from '@angular/core';
+
+// import { StoreModel } from '../../../../shared/models/store.model';
+// import { Store } from '@ngrx/store';
+
+interface VehicleCheckComponentState {
+}
+
+@Component({
+  selector: 'vehicle-check',
+  templateUrl: 'vehicle-check.html',
+})
+export class VehicleCheckComponent implements OnInit {
+
+  componentState: VehicleCheckComponentState;
+
+  selectedShowMeQuestion: boolean = false;
+
+  constructor(
+    // private store$: Store<StoreModel>
+    ) {}
+
+  ngOnInit(): void {
+  }
+
+  ngOnDestroy(): void {
+  }
+
+  onTap = () => {
+    console.log('competency tapped');
```

Comment:
Assuming these will be sorted in a later PR.

Reply:
Yes, I want to keep a stream of PRs for this vehicle check because the logic is quite complex and don't want to get lost somewhere in the details.